cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/ConstraintTest.scala

```diff
@@ -93,11 +93,11 @@ This constraint will create an accompanying index.
 //
 
 CREATE CONSTRAINT uniqueness ON (p:Person)
-       ASSERT p.age IS UNIQUE
+       ASSERT (p.firstname, p.age) IS UNIQUE
 ###
 
-Create a unique property constraint on the label `Person` and property `age` with the name `uniqueness`.
-If any other node with that label is updated or created with a `age` that
+Create a unique property constraint on the label `Person` and properties `firstname` and `age` with the name `uniqueness`.
```

Comment:
Is the name `uniqueness` the name of the property? In that case, I'd say:
```suggestion
Create a unique property constraint with the name `uniqueness` on the label `Person` and properties `firstname` and `age`.
```

Reply:
it is the name of the index

Reply:
I think almost all of these have the index name last

Reply:
I meant the name of the constraint, sorry! If this is a consistent structure, then keep it, even if it's a bit misleading.

Reply:
I think it is only on the ref card so I think we could update it for both index and constraints